### PR TITLE
[DROOLS-5878] Fix "ApplyPmmlModelCommand.isMining()" implementation

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @jiripetrlik

See https://issues.redhat.com/browse/DROOLS-5878

### CODE REVIEW

**Comment by danielezonca** [comment]
Jenkins run fdb

**Comment by danielezonca** [comment]
jenkins do fdb

**Review by danielezonca — APPROVED** [review]

File: drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java [hunk 1]
```diff
@@ -61,5 +64,16 @@ public void testHasRequestData() {
         assertEquals("ERROR-1", resultCode);
     }
 
+    @Test
```
Comment (jiripetrlik) [inline drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java:67]: Can we use a test prefix for this method. Something like testIsMining.

File: drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java [hunk 2]
```diff
@@ -61,5 +64,16 @@ public void testHasRequestData() {
...
+    public void isMining() {
+        ApplyPmmlModelCommand cmd = new ApplyPmmlModelCommand();
+        assertNull(cmd.getHasMining());
+        assertFalse(cmd.isMining());
+        cmd.setHasMining(false);
+        assertFalse(cmd.isMining());
+        cmd.setHasMining(true);
+        assertTrue(cmd.isMining());
```
Comment (jiripetrlik) [inline drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java:75]: Maybe we should test more options. Maybe I'm doing something wrong, but I've changed the logic of isMining method backwards and the test passed anyway.

File: drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java (hunk 1 above)
Comment (gitgabrio) [reply drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java:67]: @jiripetrlik 
Done

File: drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java (hunk 2 above)
Comment (gitgabrio) [reply drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java:75]: @jiripetrlik 
Hi. Can't follow exactly what you mean.

Comment (jiripetrlik) [reply drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java:75]: I reverted "ApplyPmmlModelCommand.java" and run the same test. But it did not fail. So, it seems to me that it does not test your changes. Or maybe I'm doing something wrong.

Comment (jiripetrlik) [reply drools-ruleunit/src/test/java/org/drools/ruleunit/command/pmml/ApplyPmmlModelCommandTest.java:75]: @gitgabrio 
Thank for your explanation about this offline. As logic did not change I think it is ok.

**Review by jiripetrlik — APPROVED** [review]

**Pull request merged**